spring-boot-project/spring-boot-actuator-autoconfigure/src/main/java/org/springframework/boot/actuate/autoconfigure/tracing/zipkin/ZipkinAutoConfiguration.java

```diff
@@ -56,10 +51,4 @@ PropertiesZipkinConnectionDetails zipkinConnectionDetails(ZipkinProperties prope
 		return new PropertiesZipkinConnectionDetails(properties);
 	}
 
-	@Bean
-	@ConditionalOnMissingBean
-	public BytesEncoder<Span> spanBytesEncoder() {
```

Comment:
`BytesEncoder<Span>` is only used by opentelemetry so moved it there

Reply:
I wonder if `BytesEncoder<Span>` will be useful outside of OpenTelemetry. It has been moved to `ZipkinConfigurations.OpenTelemetryConfiguration`, and I wonder if we should leave it in `ZipkinAutoConfiguration`.

Reply:
Right now, this could break users. I don't really know how many users depend on `BytesEncoder<Span>` be available, but I would leave it in `ZipkinAutoConfiguration` instead of moving it to the OpenTelemetry configuration.